front/superzol-app/src/components/register.jsx

```diff
@@ -16,6 +16,8 @@ import PersonIcon from "@mui/icons-material/Person";
 import EmailIcon from "@mui/icons-material/Email";
 import LockIcon from "@mui/icons-material/Lock";
 import AccountCircleIcon from "@mui/icons-material/AccountCircle";
+import {validatePassword} from '../utils/passwordUtils';
```

Comment:
let's keep it passwordUtils for now but if we see that we need more utils we should change the name to something more generic.